packages/image_picker/image_picker/example/integration_test/old_image_picker_test.dart

```diff
@@ -2,8 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('placeholder test', (WidgetTester tester) async {});
```

Comment:
does the `old_` prefix in the filename mean something?

Reply:
I dont think it does. It was just an empty test before. I think we just never changed the test name: https://github.com/flutter/plugins/commits/master/packages/image_picker/image_picker/example/integration_test

 I changed it to just `Image_picker_test.dart`.